app/src/renderer/vuex/modules/blockchain.js

```diff
@@ -43,18 +43,9 @@ export default ({ node }) => {
           return blockMetaInfo
         }
         state.loading = true
-        blockMetaInfo = await new Promise((resolve, reject) => {
-          node.rpc.blockchain(
-            { minHeight: String(height), maxHeight: String(height) },
-            (error, data) => {
-              if (error) {
-                reject(new Error(`Couldn't query block. ${error.message}`))
-              } else {
-                resolve(data.block_metas && data.block_metas[0])
-              }
-            }
-          )
-        })
+        blockMetaInfo = await node.rpc
```

Comment:
Opinion (personal): usually I like to use only one way of handling promises

Reply:
done